third_party/bigframes_vendored/sklearn/preprocessing/_encoder.py

```diff
@@ -54,13 +54,16 @@ class OneHotEncoder(BaseEstimator):
             print(enc.transform(bpd.DataFrame({"a": ["Female", "Male"], "b": ["1", "4"]})))
     """
 
-    def fit(self, X):
+    def fit(self, X, y=None):
         """Fit OneHotEncoder to X.
 
         Args:
             X (bigframes.dataframe.DataFrame or bigframes.series.Series):
                 The DataFrame or Series with training data.
 
+            y : None
```

Comment:
According to docstring guideline, this should be `y (None):`

Reply:
Done